app/services/github/oauth_client.rb

```diff
@@ -111,7 +111,6 @@ def faraday_middleware_stack
       # <https://github.com/octokit/octokit.rb#caching>
       # and <https://github.com/octokit/octokit.rb/blob/master/lib/octokit/default.rb>
       Faraday::RackBuilder.new do |builder|
-        builder.use Faraday::HttpCache, store: Rails.cache, serializer: Marshal, shared_cache: false
```

Comment:
I think this is the only place where we use it, so we should probably also remove it from the `Gemfile`.

Reply:
ooooh good call! Will make a follow up and see what breaks, if nothing 🔪